src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -61,10 +62,11 @@ Tabulated::Tabulated(
     }
 
     interpolators_.reserve(aContributionProfile.cols());
-    for (i = 0; i < (Index)aContributionProfile.cols(); ++i)
+    for (i = 0; i < (Index)aContributionProfile.cols(); i++)
```

Comment:
Why change this?
https://stackoverflow.com/questions/24853/what-is-the-difference-between-i-and-i

Reply:
They're effectively identical when its used like that in a loop, so its just a style choice here. But you're right, no need to change it.
```suggestion
    for (i = 0; i < (Index)aContributionProfile.cols(); ++i)
```